mafiasi/gprot/templates/gprot/list_own.html

```diff
@@ -31,6 +33,18 @@
         <span class="label label-danger">{% trans "unpublished" %}</span>
         {% endif %}
         </td>
+        <td>
+            {% for label in gprot.labels.all %}
+            <span class="label label-{{ label.colour }}">{{ label.tag }}</span>
+            {% endfor %}
```

Comment:
To prevent the redundancy, this may be moved to a different template file that can be included.